dependent/acta-veterinaria-scandinavica.csl

```diff
@@ -1,18 +1,16 @@
 <?xml version="1.0" encoding="utf-8"?>
 <style xmlns="http://purl.org/net/xbiblio/csl" version="1.0" default-locale="en-US">
-  <!-- Generated with https://github.com/citation-style-language/utilities/tree/master/generate_dependent_styles/data/bmc -->
+  <!-- Generated with https://github.com/citation-style-language/utilities/tree/master/generate_dependent_styles/data/springer -->
   <info>
     <title>Acta Veterinaria Scandinavica</title>
-    <title-short>Acta Vet Scand</title-short>
     <id>http://www.zotero.org/styles/acta-veterinaria-scandinavica</id>
     <link href="http://www.zotero.org/styles/acta-veterinaria-scandinavica" rel="self"/>
-    <link href="http://www.zotero.org/styles/biomed-central" rel="independent-parent"/>
-    <link href="http://www.actavetscand.com" rel="documentation"/>
+    <link href="http://www.zotero.org/styles/springer-basic-brackets" rel="independent-parent"/>
+    <link href="http://www.springer.com/cda/content/document/cda_downloaddocument/Key_Style_Points_1.0.pdf" rel="documentation"/>
+    <link href="http://www.springer.com/cda/content/document/cda_downloaddocument/manuscript-guidelines-1.0.pdf" rel="documentation"/>
     <category citation-format="numeric"/>
-    <category field="biology"/>
-    <category field="medicine"/>
-    <issn>1751-0147</issn>
-    <updated>2014-05-09T12:00:00+00:00</updated>
+    <eissn>1751-0147</eissn>
+    <updated>2014-05-15T12:00:00+00:00</updated>
```

Comment:
For large collections like this one, it's probably best not to touch it too often, since updating the time stamp will change **all** the files.

Plus people should retrieve their styles from https://github.com/citation-style-language/styles-distribution anyway, which always has correct time stamps.